__tests__/server.test.js

```diff
@@ -127,4 +127,45 @@ describe('GET /api/articles', () => {
             expect(errorMsg).toBe('404: Not Found')
         })
     })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with an array of comments for given article_id', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(( { body } ) => {
+            const comments = body.article
+            expect(Array.isArray(comments)).toBe(true)
+            expect(comments.length).toBe(2)
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                        body: expect.any(String),
+                        votes: expect.any(Number),
+                        author: expect.any(String),
+                        article_id: expect.any(Number),
```

Comment:
We are requesting the comments of a specific article_id - we can be more specific here!